lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -71,8 +76,13 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 	portLayerClient = client.New(t, nil)
 	portLayerServerAddr = portLayerAddr
 
+	log.Info("Creating image store")
+	if err := createImageStore(); err == nil {
+		return err
+	}
+
 	// attempt to update the image cache at startup
-	log.Info("Refreshing image cache...")
+	log.Info("Refreshing image cache")
 	go func() {
```

Comment:
PS.  This loop is just a workaround since we dont have that barrier mentioned earlier.